src/gbo/GameBoyCore.js

```diff
@@ -7895,12 +7895,20 @@ GameBoyCore.prototype.memoryReadJumpCompile = function () {
 						return 0x8F | parentObj.memory[0xFF75];
 					}
 					break;
-				case 0xFF76:
-				case 0xFF77:
-					this.memoryHighReader[index & 0xFF] = this.memoryReader[index] = function (parentObj, address) {
-						return 0;
-					}
-					break;
+                case 0xFF76:
+                    //Undocumented realtime PCM amplitude readback:
```

Comment:
These exist only in GBC mode. I think it was implemented wrong, in that we're supposed to be reading back the value after the duty cycle masking out the volume, not simply the volume.